src/App.vue

```diff
@@ -1,36 +1,38 @@
 <template>
     <div id="app">
         <Header :title="$t('app.title')" />
-        <div
-            id="site-container"
-            class="container"
-        >
-            <div class="page-head">
-                <div class="select-license-column">
-                    <h2 class="vocab h2a ha">
-                        {{ $t('select-license.heading') }}
-                    </h2>
-                    <p class="stepper-instructions vocab-body body-bigger">
-                        {{ $t('select-license.instructions') }}
-                    </p>
+        <main class="main has-background-light">
```

Comment:
Jackie, instead of adding another element, could you add background color to the `div#app` ? Adding element really adds a lot of change to git diff, and might throw off  some styling.